pkg/spec/spec.go

```diff
@@ -411,10 +412,13 @@ func (config *CreateConfig) createConfigToOCISpec(runtime *libpod.Runtime, userM
 		if !addedResources {
 			configSpec.Linux.Resources = &spec.LinuxResources{}
 		}
-		if addedResources && !cgroup2 {
-			return nil, errors.New("invalid configuration, cannot set resources with rootless containers not using cgroups v2 unified mode")
+
+		canUseResources := cgroup2 && (runtimeConfig.CgroupManager == libpod.SystemdCgroupsManager)
+
+		if addedResources && !canUseResources {
+			return nil, errors.New("invalid configuration, cannot set resources with rootless containers not using cgroups v2 and systemd")
```

Comment:
I really have no idea what this message means, even after rereading it multiple times. Is it just me? If not, could there be another way to phrase it?

Reply:
I've rephrased it in the last version